shell/protoc.sh

```diff
@@ -32,22 +32,29 @@ if [[ -z $uid ]] || [[ -z $gid ]]; then
 fi
 
 get_import_basename() {
-  basename "$(go list -f '{{ .Path }}' -m "$1" | sed 's|/v[0-9]||') " # sed removes the version off the module path if present
+  local module=$(echo $1 | jq -r .module)
+  # sed removes the version off the module path if present
+  basename $(go list -f '{{ .Path }}' -m "$module" | sed 's|/v[0-9]||')
+}
+get_import_path() {
+  local module=$(echo $1 | jq -r .module)
+  # sed removes the version off the module path if present
+  module_root=$(go list -f '{{ .Dir }}' -m "$module" | sed 's|/v[0-9]||')
+  echo "${module_root}$(echo $1 | jq -r .path)"
 }
 for import in $(get_list "go-protoc-imports"); do
+  module=$(echo $import | jq -r .module)
```

Comment:
same here:

```suggestion
  module=$(jq -r .module <<< "$import")
```

Reply:
(also quoting variables is good practice to avoid accidentally splitting on ` `)